docs/Model_Equations.rst

```diff
@@ -138,120 +154,156 @@ For the sake of simplicity, the following table gives an example for each asset
      - hp
      - kWth
 
-All grids and dispatchable sources are assumed to be available 100% of the time with no consumption limits. The MVS includes a sink component for excess energy, connected to each bus in the system and denoted by E\ :sub:`ex`\  in the equations. This excess sink accounts for the extra energy in the system that has to be dumped.
+All grids and dispatchable sources are assumed to be available 100% of the time with no consumption limits.
+The MVS includes a sink component for excess energy, connected to each bus in the system and denoted by :math:`E_{ex}` in the equations.
+This excess sink accounts for the extra energy in the system that has to be dumped.
 
 Electricity Grid Equation
 #########################
 
+The electricity grid is modeled though a feedin and a consumption node.
+Transformers limit the peak flow into or from the local electricity line.
+Electricity sold to the grid experiences losses in the transformer :math:`(ts,f)`.
+
 .. math::
         E_{grid,c}(t) - E_{grid,f}(t) + E_{ts,f}(t) \cdot \eta_{ts,f} - E_{ts,c}(t) = 0 \qquad  \forall t
         
 .. math::
-        E_{grid,c} \text{: energy consumption from the electricity grid}
+        E_{grid,c} &\text{: energy consumption from the electricity grid}
         
-        E_{grid,f} \text{: energy feed into the electricity grid}
+        E_{grid,f} &\text{: energy feed into the electricity grid}
         
-        E_{grid,c} \text{: transformer station feed-in}
+        E_{grid,c} &\text{: transformer station feed-in}
         
-        \eta_{ts,f} \text{: transformer station efficiency}
+        \eta_{ts,f} &\text{: transformer station efficiency}
         
-        E_{grid,c} \text{: transformer station consumption}
+        E_{grid,c} &\text{: transformer station consumption}
  
 Non-Dispatchable Source Equations
 #################################
 
+Non-dispatchable sources in our example are a wind, pv and solar thermal plant.
+Their generation is determined by the provided timeseries of instantaneous generation, providing :math:`\alpha`, :math:`\beta`, :math:`\gamma`.
+
 .. math::   
-        E_{wind}(t) = CAP_{wind} \cdot \alpha_{wind}(t) \qquad  \forall t
+        E_{wind}(t) &= CAP_{wind} \cdot \alpha_{wind}(t) \qquad  \forall t
         
-        E_{pv}(t) = CAP_{pv} \cdot \beta_{pv}(t) \qquad  \forall t
+        E_{pv}(t) &= CAP_{pv} \cdot \beta_{pv}(t) \qquad  \forall t
         
-        E_{stc}(t) = CAP_{stc} \cdot \gamma{stc}(t) \qquad  \forall t
+        E_{stc}(t) &= CAP_{stc} \cdot \gamma{stc}(t) \qquad  \forall t
```

Comment:
there should be changed to E_{stc}(t) &= CAP_{stc} \cdot \gamma_{stc}(t) \qquad  \forall t
there is a missing underscore